cmd/unused/internal/ui/interactive/model.go

```diff
@@ -259,3 +259,33 @@ func deleteCurrent(p deleteProgress) tea.Cmd {
 
 	return func() tea.Msg { return p }
 }
+
+func newHelp() help.Model {
+	keyStyle := lipgloss.NewStyle().Foreground(lipgloss.AdaptiveColor{
+		Light: "#909090",
+		Dark:  "#AAAAAA",
+	})
+
+	descStyle := lipgloss.NewStyle().Foreground(lipgloss.AdaptiveColor{
+		Light: "#B2B2B2",
+		Dark:  "#999999",
```

Comment:
I tried with yellow:
```suggestion
		Dark:  "#FFFF00",
```
and for me it's more readable :)
TODO: make some .rc file with colors, for now I'm fine with any option :)